44.md

```diff
@@ -0,0 +1,170 @@
+NIP-44
+======
+
+Encrypted Direct Message (Versioned)
+------------------------------------
+
+`optional` `author:paulmillr` `author:staab`
+
+The NIP introduces versioned encryption, allowing multiple algorithm choices to exist simultaneously.
+
+The algorithm described in NIP4 is potentially vulnerable to [padding oracle attacks](https://en.wikipedia.org/wiki/Padding_oracle_attack) and uses keys which are not indistinguishable from random.
```

Comment:
Delete

Reply:
I think this is helpful context for now.